public/js/admin.js

```diff
@@ -129,8 +129,7 @@ var displayQuestionForm = function() {
             $('#qType').change(function(evt) {
                 // get the answer part for the form requested
                 // id to replace is qAnswer
-                var form = "/" + $(this).val();
-                getQuestionFormAnswer(form);
+                getQuestionFormAnswer($(this).val());
```

Comment:
it seems like there are 2 PRs open that change this specific line. what is happening here?

Reply:
the same applies for all the other changes that conflict with https://github.com/quizzard-edu/quizzard/pull/22/files